sdk/python/v1beta1/kubeflow/katib/models/v1beta1_experiment_status.py

```diff
@@ -40,10 +40,12 @@ class V1beta1ExperimentStatus(object):
         'failed_trial_list': 'list[str]',
         'killed_trial_list': 'list[str]',
         'last_reconcile_time': 'datetime',
+        'metrics_unavailable_trial_list': 'list[str]',
```

Comment:
> You manually generated using gen-python-sdk/gen-sdk.sh ? 

Yes, I updated these files with `make generate`.

> This will help us to ensure that generated files are uptodate

It makes sense, thanks!

Reply:
Done.